infobip-typescript-generator-extension-api/src/main/java/com/infobip/typescript/validation/SizeToTsDecoratorConverter.java

```diff
@@ -21,12 +20,14 @@ public SizeToTsDecoratorConverter(ValidationMessageReferenceResolver validationM
     public Stream<TsDecorator> convert(Field field, Size annotation) {
         String min = String.valueOf(annotation.min());
         if (field.getType().equals(String.class)) {
-            return Stream.concat(max("MaxLength", annotation, annotation.max()),
-                                 getMin(field, annotation, min));
+            return Stream.of(getMax("MaxLength", annotation, annotation.max()),
```

Comment:
Exactly. This is needed because Java `@Size` passes validation for nulls, but Typescript `@MaxLength` and `@MinLength` do not allow null or undefined.